src/Features/CSharp/Portable/Completion/KeywordRecommenders/AnnotationsKeywordRecommender.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using Microsoft.CodeAnalysis.CSharp.Extensions.ContextQuery;
+
+namespace Microsoft.CodeAnalysis.CSharp.Completion.KeywordRecommenders
+{
+    internal class AnnotationsKeywordRecommender : AbstractSyntacticSingleKeywordRecommender
```

Comment:
>AnnotationsKeywordRecommender [](start = 19, length = 29)

nice. You'll want to add some IDE tests for this change (recommenders and classification fix)

Reply:
Done